[API](../README.md) / [local-peers](../modules/local_peers.md) / PeerFailedConnectionError

# Class: PeerFailedConnectionError

[local-peers](../modules/local_peers.md).PeerFailedConnectionError

## Hierarchy

- `Error`

  ↳ **`PeerFailedConnectionError`**

## Table of contents

### Constructors

- [constructor](local_peers.PeerFailedConnectionError.md#constructor)

### Properties

- [name](local_peers.PeerFailedConnectionError.md#name)

## Constructors

### constructor

• **new PeerFailedConnectionError**(`message?`): [`PeerFailedConnectionError`](local_peers.PeerFailedConnectionError.md)

#### Parameters

| Name | Type |
| :------ | :------ |
| `message?` | `string` |

#### Returns

[`PeerFailedConnectionError`](local_peers.PeerFailedConnectionError.md)

#### Overrides

Error.constructor

#### Defined in

[src/local-peers.js:605](https://github.com/digidem/mapeo-core-next/blob/53dc843a45bb963f7a880f5f7973107d5b1fb99c/src/local-peers.js#L605)

## Properties

### name

• **name**: `string`

#### Inherited from

Error.name

#### Defined in

[src/local-peers.js:607](https://github.com/digidem/mapeo-core-next/blob/53dc843a45bb963f7a880f5f7973107d5b1fb99c/src/local-peers.js#L607)
